docs/modules/ROOT/pages/admin.adoc

```diff
@@ -85,6 +85,23 @@ Custom actions can also be _repeated_, which will present you to a pre-filled fo
 
 WARNING: Certain ABI types, such as nested structs, are not yet supported. Contact us if you need to call a function that is currently unsupported!
 
+[[batches]]
+=== Batches
+
+A _batch proposal_ is a call to multiple functions from different contracts. It allows you to atomically execute multiple actions in a single transaction.
```

Comment:
Nit, but not necessarily _different_ contracts right?

Reply:
We clarify that there is a restriction to batches only working for calls on the same network

Reply:
Both are right. Not necessarily different contracts, and on the same network.
Let me add those notes. Thanks @shahnami!